CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## [Unreleased]
 
+### Added
+
+- Added a `prepare_release` command to the `devtool` utility that updates the
```

Comment:
This looks like it deserves a development doc. I do not want to block merging the commit on that, so I recommend adding an issue for creating a Development Guide document in which we explain this sort of processes.

Reply:
Logged #905.